get-started/faq/index.md

```diff
@@ -126,8 +126,8 @@ Fungibility is a simple property of money such that there are no differences bet
 </div>
 
 <div class="tab">
-    <input id="tab-eleven" type="checkbox" name="tabs" class="accordian">
-    <label for="tab-eleven" class="accordian">If Monero is so private how do we know they're not being created out of thin air?</label>
+    <input id="tab-eleven" type="checkbox" name="tabs" class="accordion">
```

Comment:
This needs to be `tab-twelve` otherwise it will collapse/expand the above tab (below too).